lumen-api/Jobs_Nabu_PostGIS/ci/k8s/cronjob.yaml.template

```diff
@@ -3,7 +3,7 @@ kind: CronJob
 metadata:
   name: nabu-postgis
 spec:
-  schedule: "* */2 * * *"
+  schedule: "@hourly"
```

Comment:
Given an hourly schedule, with an allow concurrencyPolicy, a backoffLimit of 6 and an activeDeadlineSeconds of 15 mins, it means that it maybe two jobs will end up running at the same time. Is that ok?

Reply:
Given the velocity of data i think for starters , 0 0 * * 0 will suffice

Reply:
@dlebrero my code was a _skeleton_ I didn't have a the actual business requirement in how often the data needs to be refreshed. I wanted to test that the code/cronjob works with frequent updates.